src/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is there by chance a package for this font that we could depend on instead of bloating git?

Reply:
The best we can do it only use the font we need.